srv/src/main/resources/application.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 spring:
-  resources.static-locations: "file:./app"
+  web:
```

Comment:
Thanks. Can you keep the single-line definition to make the configuration file a little more compact?